src/Symfony/Component/Form/FormErrorIterator.php

```diff
@@ -266,6 +267,27 @@ public function seek($position)
     }
 
     /**
+     * Creates iterator for errors with specific codes.
+     *
+     * @param string|string[] $codes The codes to find
+     *
+     * @return static New instance which contains only specific errors.
+     */
+    public function findByCodes($codes)
```

Comment:
I haven't see any benefits from public method `findByCause` - too much code for writing closure. It equals to writing foreach.